cit3.0-web/src/data/userStoryOptions2.json

```diff
@@ -0,0 +1,39 @@
+{
+    "userStoryOptions2": [
+      {
+        "id": "1",
+        "code": "VIEWING",
+        "label": "viewing Investment Opportunity matching my criteria",
+        "text1": "We have a tool you might like to check out: our CIOT. (description). Would you like to go to the CIOT?",
+        "text2": null
+      },
+      {
+        "id": "2",
+        "code": "LEARNING",
+        "label": "learning more about the province or a specific area of BC",
+        "text1": "<h3>Great!</h3> <p>You can choose between different types of areas from Communities and Unincorporated Areas, First Nations land and Reserves, Regional District to larger Economic Regions or Tourism Regions.</p><p>What kind of area would you like to look at?</p>",
+        "text2": "I would like to like at"
```

Comment:
Is this text correct? I dont know the whole context but Im having a hard time trying to figure out where this would make sense haha

Reply:
this is correct, part of create your own story text